ERCS/erc-7835.md

```diff
@@ -0,0 +1,250 @@
+---
+eip: 7835
+title: Universal On-Chain Metadata Parser Interface
+description: A universal interface for parsing wallet-accessed metadata, enhancing security, usability, and data readability.
```

Comment:
Your description contains only either a restatement of your title ("A universal interface for parsing...metadata") or fluff ("enhancing security, usability, and data readability".)

Try to include _more_ information in your description than was already introduced in your title.

For example, you could try to answer questions like:

- What is metadata?
- How is it universal?
- How is it parsed on chain, and why would you want it there?